pages/common/sleep.md

```diff
@@ -14,3 +14,11 @@
 - Delay in hours:
 
 `sleep {{hours}}h`
+
+- Pause for 1 day 3 hours:
```

Comment:
```suggestion
- Delay for a specific amount of time (in d|h|m|s):
```
How about this?

Reply:
This example is to show multiple parameters with different units can also be used. To mention all the units I've edited line 10.